python/orca/test/bigdl/orca/learn/ray/tf/test_tf2estimator_ray_backend.py

```diff
@@ -100,7 +183,7 @@ def get_estimator(workers_per_node=2, model_fn=model_creator):
     return estimator
 
 
-class TestTF2EstimatorTF2Backend(TestCase):
+class TestTF2EstimatorRayBackend(TestCase):
     def setUp(self):
         self.sc = init_orca_context(init_ray_on_spark=True)
```

Comment:
We already have `init_orca_context` in `conftest.py`, would this cause conflict? or is it better to add the parameter `init_ray_on_spark=True` in `conftest.py`?